### DISCUSSION TITLE
Can we use Tiptap without Vue?

### DISCUSSION THREAD

**Discussion by l2aelba** [ueberdosis/tiptap#958] [General Questions]
A bit wired question but I cannot find any resource about that.

Im not sure how to run **Tiptap without Vue** 

Can we do something like ... ?

```js
import { Editor } from 'tiptap'
new Editor({
    content: '',
    selector: '#my-id' // or ".my-class"
})
```

Thanks.

**Comment by Priestch** [comment]
@l2aelba If you don't want the `Vue` related feature, then you can use your bundle tool(webpack or rollup) to fake the `vue` package, it just works.

**Comment by hanspagel** [comment]
Thanks @Priestch for chiming in!

@l2aelba: Or you can sponsor us and use tiptap v2. 😅
